compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -502,6 +502,18 @@ static bool canUseInOperandAsInitOperand(OpOperand *inOperand,
   return true;
 }
 
+/// All operations in a dispatch should be vectorized, which isnt the case today
+/// This is an explicit list of operations that arent vectorized for now
+/// requiring special handling for now in dispatch region formation to avoid
+/// large stack allocations.
+static bool isVectorizedAlways(Operation *producer) {
+  if (auto convOp = dyn_cast<linalg::Conv2DNhwcHwcfOp>(producer)) {
```

Comment:
I think for most backends we can vectorize this... I think if we can make this work on RoCM then we can use that to drive this forward....

Reply:
I think we should hold the bar for fusion. We don't disable fusion just because one of the backends does not support it. The PR is probably fine because the number of dispatches is improved (though it may comes from other changes). Can you file an issue, assign it to someone, and add a TODO with the issue number?

Reply:
The fusion changes have a chicken-and-egg effect. You need the backends to handle it before we can turn it on, and backend fixes dont happen without it being stressed due to fusion. So the best way forward is to land the fusion heuristic changes under a flag and get the backend to fix it. For now filed https://github.com/iree-org/iree/issues/17167